renderer.go

```diff
@@ -157,22 +182,29 @@ func (r *Renderer) drawCheckTile() {
 
 func (r *Renderer) drawBoard(o Options) error {
 	for _, position := range r.board {
-		if err := r.drawPiece(position, o.AssetPath, o.Resizer); err != nil {
+		if err := r.drawPiece(position, o.AssetPath, o.Resizer, o.Inverted); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
 func (r *Renderer) drawRankFile(o Options) error {
+	var symbols string
 	fontPath, err := findfont.Find("arial.ttf")
 	if err != nil {
 		return err
 	}
 	if err := r.context.LoadFontFace(fontPath, 14); err != nil {
 		return err
 	}
-	for i, symbol := range fileSymbols {
+
+	if o.Inverted {
+		symbols = reverse(fileSymbols)
```

Comment:
@ZurabWeb I'm not sure if the compiler optimizes this, but it would be better to have a defined reversed fileSymbols variable hardcoded so it doesn't have to constantly reverse this string at runtime.